src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -159,4 +216,24 @@ public double getShoulderAngle() {
 	public double getWristAngle() {
 		return wristEncoder.getDistance();
 	}
+
+	public PositionType getPosition() {
+		return currentPosition;
+	}
+
+	/* TODO:
+	 * FeedForward for Arm and Wrist
+	 */
+	@Override
+	public void periodic() {
+		if (!manualOverride) {
+			// for preset angle mode
+			armMotor.setVoltage(calculateArmPID() + calculateArmFeedforward());
+			wristMotor.setVoltage(calculateWristPID() + calculateWristFeedforward());
+		} else {
+			// for manual mode
+			armMotor.setVoltage(calculateWristFeedforward());
+			wristMotor.setVoltage(calculateArmFeedforward());
```

Comment:
This looks like it would conflict with the `setArmMotor()` and `setWristMotor()` methods (also I don't think we would use the feedforward in manual mode).